spec/features/merchants/merchant_invoice_show_page_spec.rb

```diff
@@ -29,5 +30,12 @@
           expect(page).to have_content(@invoice1.date_format)
         end
     end
+    it "I see all of the customer information related to that invoice " do
+      visit "/merchant/#{@merchant.id}/invoices/#{@invoice1.id}"
+        within "#invoice-customer-info" do
+          expect(page).to have_content(@customer.name)
+          expect(page).to_not have_content(@not_customer.name)
```

Comment:
I like that we are checking to make sure only one customer name is expected for that customer's invoice!